sale_automatic_workflow_payment/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,6 @@
+This modules allows to define an automatic workflow to a Sale Order when a
+payment transaction is created for it (to manage automatic invoice creation or
+...).
+
+Then, when the transaction is done, let the Odoo standard process flow
+to create corresponding payments and reconciliation (through a cron).
```

Comment:
@rousseldenis Is this cron active by default? If not, we should mention that the cron must be activeted

Reply:
Yes! See: https://github.com/odoo/odoo/blob/13.0/addons/payment/data/payment_cron.xml#L8

Reply:
I've added more info